### PR TITLE
Personal portfolio

### PR DESCRIPTION
**Pull request by mprotic123**
This pull request is created in order to review an assignment from Microverse.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        <a href="#"
+          ><img class="icon" src="icons/medium.svg" alt="medium logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/github.svg" alt="github logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/angellist.svg" alt="angellist logo"
+        /></a>
+      </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag), nice to give a link to each icon, please use ul and li to align your icons as it is required.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+      <nav class="menu">
+        <button type="button" class="btn">
+          <i class="material-icons">menu</i>
+        </button>
+      </nav>
+    </header>
+    <section class="headline background">
+      <div>
+        <h1>Hey there. I’m Milan</h1>
+        <h2>I’m a software developer</h2>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Please change h2 to p tag as it is required in the project requirements.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
```
Comment (Emmyn5600) [inline index.html]: - [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+            </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
```
Comment (Emmyn5600) [inline index.html]: - [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+            </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">JavScript</a>
+              </li>
+              <li class="card_btn tag_btn">
+                <a href="#" class="tag_txt">html</a>
+              </li>
+            </ul>
+
+            <div class="card_btn act_btn">
+              <a href="#" class="act_txt">See Project</a>
```
Comment (Emmyn5600) [inline index.html]: - [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
```
Comment (Emmyn5600) [inline index.html]: - [ ] The Figma template provided styling rules for hover, active (when it's pressed), and enabled (default state). Kindly implement this requirement.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @mprotic123 

Good job so far!:clap::clap:
You have done well in implementing the project requirements:clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

May I highlight the following:
- The repo looks professional:heavy_check_mark:
- Commit messages are good and descriptive:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:

##  Required Changes:writing_hand:

* Kindly, Include a live demo link in your Readme file.

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**